.github/workflows/publish.yml

```diff
@@ -0,0 +1,87 @@
+name: publish artifacts
+on:
+  push:
+    branches:
+      - 'main'
+  workflow_dispatch:
+jobs:
+  build-napi-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: napi build
+        run: yarn install && yarn build
```

Comment:
does `napi build` not build any other artifacts? e.g. typescript contracts, README for the target package?

Reply:
- `napi build` only creates `index.d.ts` and `index.js` files
- `napi build --platform` creats the files mentioned above + *.node files needed to run.
- `yarn build` calls `napi build --platform`